Makefile

```diff
@@ -51,19 +51,19 @@ use-oncluster-controller:
 
 # Install CRDs into a cluster
 install: manifests
-	kubectl apply -f config/crds
+	kubectl apply -f config
 
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
 deploy: manifests
-	kubectl apply -f config/crds
+	kubectl apply -f crds
```

Comment:
I don't have a `crds` dir after running make manifests. Not sure what this is doing.

Reply:
```
make deploy
/Users/dwhatley/code/go/bin/controller-gen "crd:trivialVersions=true" crd rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crds/bases 
kubectl apply -f crds
error: the path "crds" does not exist
make: *** [deploy] Error 1

```

Reply:
Oh, wait, this is wrong. It should be "config" and not "crds".

Reply:
config/crds no longer has the CRDs. They're now one level up in "config".